src/java.base/aix/classes/sun/nio/ch/AixPollPort.java

```diff
@@ -303,11 +307,23 @@ private class EventHandlerTask implements Runnable {
         private Event poll() throws IOException {
             try {
                 for (;;) {
-                    int n;
+                    int n, m;
```

Comment:
I'd declare `m` in the `try` block. It's not used outside of it.

Reply:
Done